drivers/dotnet-getting-started.md

```diff
@@ -3,115 +3,16 @@ layout: default
-        {
-            Name = "arangodb-net-standard",
-            Users = new List<DatabaseUser>
-            {
-                new DatabaseUser
-                {
-                    Username = "jlennon",
-                    Passwd = "yoko123"
-                }
-            }
-        });
-}
-```
+## Reference
 
-Since the system database is only used once to create another separate database, `systemDbTransport` is wrapped in a `using` block.
-
-In general, if you connect to the same database a lot, you don't want to dispose `HttpApiTransport` until the end of your application's life.
-
-### Create a collection
-
-```csharp
-// Use your new database with basic auth credentials for the user jlennon.
-var transport = HttpApiTransport.UsingBasicAuth(
-    new Uri("http://localhost:8529"),
-    "arangodb-net-standard",
-    "jlennon",
-    "yoko123");
-var adb = new ArangoDBClient(transport);
-// Create a collection in the database
-await adb.Collection.PostCollectionAsync(
-    new PostCollectionBody
-    {
-        Name = "MyCollection"
-        // A whole heap of other options exist to define key options, 
-        // sharding options, etc
-    });
-```
-
-### Create documents
-
-```csharp
-// Create document in the collection using anonymous type
-await adb.Document.PostDocumentAsync(
-    "MyCollection",
-    new
-    {
-        MyProperty = "Value"
-    });
-// Create document in the collection using strong type
-await adb.Document.PostDocumentAsync(
-    "MyCollection",
-    new MyClass
-    {
-        ItemNumber = 123456,
-        Description = "Some item"
-    });
-```
-{% hint 'tip' %}
-The document object must not have any value against a property named `_key`, if you expect ArangoDB to generate the document key for you.
-The default serializer options specify that null values will be ignored, so if your class has a `_key` property, you can leave it as `null` when creating a new document.
-If you change the serializer options so that `IgnoreNullValues` is `false`, then you cannot create a new document using a class that specifies a property named `_key`, because the ArangoDB API will reject the request.
-{% endhint %}
-
-
-### Run an AQL query
-
-```csharp
-// Run AQL query (create a query cursor)
-var response = await adb.Cursor.PostCursorAsync<MyClassDocument>(
-    @"FOR doc IN MyCollection 
-      FILTER doc.ItemNumber == 123456 
-      RETURN doc");
-MyClassDocument item = response.Result.First();
-```
-
-### Patch a document
-
-```csharp
-// Partially update document
-await adb.Document.PatchDocumentAsync<object, object>(
-    "MyCollection",
-    item._key,
-    new { Description = "More description" });
-```
-
-### Replace a document
-
-```csharp
-// Fully update document
-item.Description = "Some item with some more description";
-await adb.Document.PutDocumentAsync(
-    $"MyCollection/{item._key}",
-    item);
-```
+Browse the full reference for [ArangoDBNetStandard](dotnet-reference/Home.md).
```

Comment:
Where is this supposed to point to?

Reply:
Hi @Simran-B , thanks for your feedback. There is a reference documentation of the driver that is generated by DocFx. The idea was to create a subfolder dotnet-reference where Home.md would be the main page of that reference documentation. I deleted the reference documentation because it was too big (over 2000 md pages). I will remove the reference to it here for now and we will put it back in once we decide on where to place the reference documentation.

Reply:
Generated reference documentation should be hosted through GitHub Pages, with the option to fully automate the generation and deployment. For reference, see how [arangojs](https://github.com/arangodb/arangojs) does it (hosted site: https://arangodb.github.io/arangojs/).

Reply:
We're working on pushing the reference to Github Pages. Once that's done, I will update the link here to point to it.

Reply:
Hi @Simran-B , I have successfully published the reference docs and added it's link to the Getting Started page. Kindly review. Thanks